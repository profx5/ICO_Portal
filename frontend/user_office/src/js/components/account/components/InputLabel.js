import React from 'react'
import styled from 'styled-components';


const InputLabel = ({children, htmlFor}) => {

    return (
        <StyledLabel htmlFor={htmlFor}>
            {children}
        </StyledLabel>
    );
}


export default InputLabel;

const StyledLabel = styled.label`
    font-size: 16px;
    color: #0a0a0a;
    display: block;
    margin-bottom: 13px;
`;
