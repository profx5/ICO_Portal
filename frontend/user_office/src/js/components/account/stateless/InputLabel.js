import React from 'react'
import styled from 'styled-components';
import {media} from 'js/services/media';


const InputLabel = ({children, htmlFor}) => {

    return (
        <StyledLabel htmlFor={htmlFor}>
            {children}
        </StyledLabel>
    );
}


export default InputLabel;

const StyledLabel = styled.label`
    font-size: 16px;
    color: #0a0a0a;
    display: block;
    margin-bottom: 13px;
    ${media.xs} {
        font-size: 12px;
    }
`;
