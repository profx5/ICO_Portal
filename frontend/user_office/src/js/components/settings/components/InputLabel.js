import React from 'react'
import styled from 'styled-components';


const InputLabel = ({children}) => {

    return (
        <StyledLabel>
            {children}
        </StyledLabel>
    );
}


export default InputLabel;

const StyledLabel = styled.label`
    font-size: 16px;
    color: #0a0a0a;
    display: block;
    margin-bottom: 13px;
`;
