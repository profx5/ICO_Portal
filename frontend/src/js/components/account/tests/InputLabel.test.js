import React from 'react';
import {configure, shallow} from 'enzyme';
import jest from 'jest-mock';
import Adapter from 'enzyme-adapter-react-16';

import InputLabel from 'js/components/account/stateless/InputLabel';


configure({ adapter: new Adapter() });

it('should render stateless InputLabel', () => {
    const props = {
        onClickHandler: jest.fn()
    }
    const component = shallow(<InputLabel {...props}/>);
    expect(component).toMatchSnapshot();
})
