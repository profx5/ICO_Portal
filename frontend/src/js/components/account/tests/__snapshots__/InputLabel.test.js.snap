// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render stateless InputLabel 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <InputLabel
    onClickHandler={[MockFunction]}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": undefined,
      "htmlFor": undefined,
    },
    "ref": null,
    "rendered": null,
    "type": Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
    font-size: 16px;
    color: #0a0a0a;
    display: block;
    margin-bottom: 13px;
    ",
          "@media (max-width: 767px)",
          " {
        font-size: 12px;
    }
",
        ],
      },
      "displayName": "styled.label",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "label",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    },
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": undefined,
        "htmlFor": undefined,
      },
      "ref": null,
      "rendered": null,
      "type": Object {
        "$$typeof": Symbol(react.forward_ref),
        "attrs": Array [],
        "componentStyle": ComponentStyle {
          "componentId": "sc-bdVaJa",
          "isStatic": true,
          "rules": Array [
            "
    font-size: 16px;
    color: #0a0a0a;
    display: block;
    margin-bottom: 13px;
    ",
            "@media (max-width: 767px)",
            " {
        font-size: 12px;
    }
",
          ],
        },
        "displayName": "styled.label",
        "foldedComponentIds": Array [],
        "render": [Function],
        "styledComponentId": "sc-bdVaJa",
        "target": "label",
        "toString": [Function],
        "warnTooManyClasses": [Function],
        "withComponent": [Function],
      },
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
